DEHPCommon/UserInterfaces/Views/Login.xaml

```diff
@@ -115,7 +136,7 @@
 
             <Button Grid.Column="2" Height="Auto" MinWidth="100" Margin="5" HorizontalAlignment="Right" Command="{Binding LoginCommand}" Width="117"
                     Visibility="{Binding Path=LoginSuccessfull, Converter={dxmvvm:BooleanToVisibilityConverter Inverse=True}, UpdateSourceTrigger=PropertyChanged}">
-                    <!--Visibility="{Binding Path=LoginSuccessfull, Converter={dxmvvm:BooleanToVisibilityConverter}, ConverterParameter={dx:VisibilityInverseConverter}, UpdateSourceTrigger=PropertyChanged}">-->
+                <!--Visibility="{Binding Path=LoginSuccessfull, Converter={dxmvvm:BooleanToVisibilityConverter}, ConverterParameter={dx:VisibilityInverseConverter}, UpdateSourceTrigger=PropertyChanged}">-->
```

Comment:
Can you remove commented code please.

Reply:
Hi, sorry for the intromission, but Alex told me I could participate with comments on this project.

On this xaml I see `Visibility="{Binding Path=LoginSuccessfull` but in the sources there is this change at `DEHPCommon/UserInterfaces/ViewModels/Interfaces/ILoginViewModel.cs`

From `public bool LoginSuccessfull`
To `public bool LoginSuccessful`

Is it the binding working?

Reply:
No it doesn't work, I should have updated the xaml with the new property name. Thanks for your comment @jpopeneng!